python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -114,9 +114,9 @@ def make_final_proxy_type(
     )
 
 
-def make_intermediate_proxy_type(name, fast_type, slow_type):
+def make_intermediate_proxy_type(name, fast_type, slow_type, **kwargs):
```

Comment:
Is `**kwargs` supposed to be `additional_attributes` now? Also which objects got impacted by this change (the diff doesn't make it immediately obvious)?

Reply:
This was a stale change, cleaned it up. Thanks Matthew!